
USB_FS_MAX_FRAMES_PRE_SCALE is only available with
kfreebsd-kernel-headers >= 8.3

--- a/lib/libusb/libusb20_ugen20.c
+++ b/lib/libusb/libusb20_ugen20.c
@@ -747,7 +747,7 @@
 	struct usb_fs_endpoint *fsep;
 
 	if (pre_scale)
-		MaxFrameCount |= USB_FS_MAX_FRAMES_PRE_SCALE;
+		MaxFrameCount |= (1U << 31);
 
 	memset(&temp, 0, sizeof(temp));
 
